virttest/iscsi.py

```diff
@@ -208,6 +208,7 @@ def __init__(self, params, root_dir):
         self.target = params.get("target")
         self.export_flag = False
         self.luns = None
+        self.lun_attrs = params.get("lun_attrs")
```

Comment:
Suggest using `iscsi_lun_attrs` as the param name.

Reply:
updated